datamodel/referencedata.d/modeoftransportcodes.csv

```diff
@@ -0,0 +1,10 @@
+Mode of Transport Code,Mode of Transport Name,Mode of Transport Description,DCSA Transport Type

```

Comment:
Vi agreed in the chat only to include the ones that have DCSA values: 1, 2, 3 and 8